CMakeLists.txt

```
@@ -162,12 +162,22 @@ APPLY_DEFAULT_APPLE_CONFIGURATION()
 
 option(BUILD_PYTHON_INTERFACE "Build the python bindings" ON)
 cmake_dependent_option(
-  GENERATE_PYTHON_STUBS
-  "Generate the Python stubs associated to the Python library"
-  OFF
+  COAL_PYTHON_NANOBIND
+  "Build the nanobind-based bindings."
+  ON
```

```suggestion
  OFF
```

Activating this by default would be an unnecessary breaking change. We can do that on the next major release.